Implement a first version of the Trust Mechanism

RensDur opened the pull request:
Files changed:
- OfficialAgent.py
- WorldBuilder.py

Please take a look at the ```_trustBelief``` function.

RensDur committed:
Implement a first version of Trust Mechanism

RensDur committed:
Add .idea and logs to gitignore

RensDur committed:
Update trust mechanism

'Search: y' directly after 'Search: x' will no longer receive as large of a penalty

julievm commented on agents1/OfficialAgent.py:
```diff
@@ -799,13 +799,130 @@ def _trustBelief(self, members, trustBeliefs, folder, receivedMessages):
+                increaseCompetenceBelief()
+
+            if 'Rescue alone' in message:
+                increaseWillingnessBelief(0.25)
```
Decrease willingness when the human tells the robot to rescue the victim alone

RensDur committed:
Update .gitignore

RensDur committed:
Add state container per received message

Messages, received from the human, can now directly be associated to a state the system was in at the point of receiving that message. This allows, for instance, for recording the time at which a message was received.

RensDur committed:
Update .gitignore

RensDur committed:
Merge pull request #3 from RensDur/impl-message-state-connection

Implement message state connection

vincentvvliet committed:
Fix merge conflicts
